### ISSUE TITLE
columnChooser prototype error

### ISSUE THREAD

**Issue by gberenfield** [tonytomov/jqGrid#722]
When I use the columnChooser(), I'm getting a `Cannot read property 'prototype' of undefined` error in the browser console.

It looks like the following section is where the error is located:
```
return ($.ui.multiselect.prototype && $elem.data($.ui.multiselect.prototype.widgetFullName || $.ui.multiselect.prototype.widgetName)) ||
					$elem.data("ui-multiselect") || $elem.data("multiselect");
```

**Comment by tonytomov** [comment]
Hello,

Please provide a test case with full code and javascript modules used?
Thank you.

Kind Regards,
Tony

**Comment by gberenfield** [comment]
Sorry, this has been fixed in 5.0.1, closing
